mac/KeymanEngine4Mac/KeymanEngine4Mac/KME/OnScreenKeyboard/OSKView.m

```diff
@@ -159,7 +186,22 @@ - (NSArray *)oskLayout {
                          [[OSKKey alloc] initWithKeyCode:MVK_QUOTE caption:@"'" scale:1.0],
                          [[OSKKey alloc] initWithKeyCode:MVK_ENTER caption:@"↵" scale:1.75], nil];
         
-        NSArray *row4 = [NSArray arrayWithObjects:
+        NSArray *row4 = [self use102ndKey] ?
+            [NSArray arrayWithObjects:
```

Comment:
Is there really no way to reuse most of the defined array? This feels pretty WET.

Reply:
`NSArray` isn't mutable but I could probably construct it with two separate arrays and concatenate. Agree this is WET but it's also easy to maintain ;-)